internal/fields/_static/allowed_geo_ips.txt

```diff
@@ -8,3 +8,23 @@
 89.160.20.128/25
 67.43.156.0/24
 2a02:cf40::/29
+1.1.1.1/32
+1.1.1.2/32
+1.1.1.3/32
+1.0.0.1/32
+1.0.0.2/32
+1.0.0.3/32
+2606:4700:4700::1111/128
+2606:4700:4700::1112/128
+2606:4700:4700::1113/128
+2606:4700:4700::1001/128
+2606:4700:4700::1002/128
+2606:4700:4700::1003/128
+2606:4700:4700::64/128
+2606:4700:4700::6400/128
+8.8.8.8/32
+8.8.4.4/32
+2001:4860:4860::8888/128
+2001:4860:4860::8844/128
+2001:4860:4860::64/128
+2001:4860:4860::6464/128
```

Comment:
Are these IPs included in the current GeoIP database? At least some of them are not related to documentation.

The idea of this list is to have a reference of IP ranges that are included in the GeoIP database, so package developers can know when GeoIP enrichment is working.

Reply:
Yeap, from: https://www.maxmind.com/en/geoip-demo

IP Address | Country Code | Location | Network | Postal Code | Approximate Coordinates* | Accuracy Radius (km) | ISP | Organization | Domain | Metro Code
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
2001:4860:4860::8888 | US | United States,North America | 2001:4860:4860::/46 |   | 37.751,-97.822 | 100 | Google | Google |   |  
2001:4860:4860::8844 | US | United States,North America | 2001:4860:4860::/46 |   | 37.751,-97.822 | 100 | Google | Google |   |  
2001:4860:4860::64 | US | United States,North America | 2001:4860:4860::/46 |   | 37.751,-97.822 | 100 | Google | Google |   |  
2001:4860:4860::6464 | US | United States,North America | 2001:4860:4860::/46 |   | 37.751,-97.822 | 100 | Google | Google |   |  